app/Models/Shops.php

```diff
@@ -4,10 +4,17 @@
 
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use App\Models\Reviews;
+
 
 class Shops extends Model
```

Comment:
前回見落としていたのですが、モデルのクラス名は単数形が良いですね
(ファイル名も同様)
```suggestion
class Shop extends Model
```